packages/block-library/src/post-comments/edit.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useEntityId } from '@wordpress/core-data';
+
+function PostCommentsDisplay( { postId } ) {
+	return useSelect(
+		( select ) => {
+			const comments = select( 'core' ).getEntityRecords( 'root', 'comment', {
+				post: postId,
+			} );
+			return (
+				comments &&
+				comments.map( ( comment ) => <p key={ comment.id }>{ comment.content.raw }</p> )
```

Comment:
Just curiosity: It is the first time I see useSelect returning directly the elements (p tags), any advantage in following this practice when compared to what we have been doing (useSelect would just return the array of elements and the mapping to elements would happen outside useSelect)?

Reply:
Not really.

I guess it's more performant because we call `createElement` less.